packages/cli/src/generate/generators/app/create-app.ts

```diff
@@ -21,6 +21,46 @@ function isYarnInstalled() {
   }
 }
 
+function validateProjectName(name: string) {
+  let flag = 0;
+  const specialChars = ['!',
+    '@',
+    '#',
+    '$',
+    '%',
+    '^',
+    '&',
+    '*',
+    '(',
+    ')',
+    '/',
+    '\\',
+    '+',
+    '{',
+    '}',
+    '[',
+    ']',
+    ':',
+    ';',
+    '<',
+    '>',
+    ',',
+    '.',
+    '?'
+  ];
+
+  specialChars.map(char => {
```

Comment:
I think we could use the method `find` here to get a more concise code. Maybe something like `return !specialChars.find(char => name.includes(char))`. This way, we could get rid of the `flag` variable too.